packages/ext-angular-modern-boilerplate/src/app/Home/home.component.ts

```diff
@@ -0,0 +1,59 @@
+import { Component, OnInit, ViewChild, ElementRef } from '@angular/core';
+import {gridData} from './data';
+
+declare var Ext: any;
+
+@Component({
+  selector: 'app-home',
+  templateUrl: './home.component.html',
+  styles: [``]
+})
+export class HomeComponent implements OnInit {
+
+  constructor() { }
```

Comment:
Could you remove the new lines and properly indent the class members?

Reply:
Fixed.